src/test/java/http/handlers/auth/AuthHandlerTest.java

```diff
@@ -0,0 +1,41 @@
+package http.handlers.auth;
+
+import http.request.HttpRequest;
+import http.request.error.InvalidRequestException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.UnsupportedEncodingException;
+import java.util.Base64;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class AuthHandlerTest {
+
+    private AuthHandler authHandler;
+    private HttpRequest httpRequest;
+
+    @BeforeEach
+    void setUp() throws InvalidRequestException, UnsupportedEncodingException {
```

Comment:
There is a lot of setup here just to get a HttpRequest object. You should not have to be doing complex things like encoding a string into byte in your test.

Reply:
I'll simplify the AuthHandler parameters so both the AuthHandler and it's test are easier to reason about.